### PR TITLE
Correct errors and expand labels for clarity

### PR DESCRIPTION
**Pull request by Hwurzburg**
since there is no CI, my typing dyslexia needs to be reviewed carefully

### CODE REVIEW

**Comment by nkruzan** [comment]
@Hwurzburg looks ok to me. Might also capitalize AHRS under Other "Enable External ahrs"...(or remove it from other as it's redundant)

**Comment by Hwurzburg** [comment]
removed it

File: app.py [hunk 1]
```diff
@@ -79,8 +79,8 @@ class Feature:
     Feature('Ident', 'ADSB_UAVIONIX', 'HAL_ADSB_UAVIONIX_MAVLINK_ENABLED', 'Enable Uavionix ADSB', 0, 'ADSB'),
     Feature('Ident', 'AIS', 'HAL_AIS_ENABLED', 'Enable AIS', 0, None),
 
-    Feature('Telemetry', 'CRSF', 'HAL_CRSF_ENABLED', 'Enable CRSF', 0, None),
-    Feature('Telemetry', 'CRSFText', 'HAL_CRSF_TEXT_SELECTION_ENABLED', 'Enable CRSF Text Selection', 0, 'CRSF'),
+    Feature('Telemetry', 'CRSF', 'HAL_CRSF_TELEMETRY_ENABLED', 'Enable CRSF Telemetry', 0, None),
```
Comment (tridge) [inline app.py:82]: i think this is HAL_CRSF_TELEM_ENABLED

File: app.py [hunk 2]
```diff
@@ -79,8 +79,8 @@ class Feature:
...
+    Feature('Telemetry', 'CRSFText', 'HAL_CRSF_TEXT_SELECTION_ENABLED', 'Enable CRSF Text Param Selection', 0, 'CRSF'),
```
Comment (tridge) [inline app.py:83]: should be HAL_CRSF_TELEM_TEXT_SELECTION_ENABLED

**Review by tridge — CHANGES_REQUESTED** [review]

**Review by tridge — APPROVED** [review]

**Pull request merged**